### DISCUSSION TITLE
Shapes like Prusa Slicer

### DISCUSSION THREAD

**Discussion by 11B4P** [OrcaSlicer/OrcaSlicer#347] [Ideas]
I noticed in both Studio and Soft Fever that no Shapes directory exists under resources. Anyway that feature can be added to Studio or SF? It's a handy feature I used often in Prusa Slicer. Not a show stopper and probably low hanging fruit, just something I think would be nice to have versus just simple squares, triangles and cylinders. I did search prior to posting! 

Thanks!

**Comment by rideorroot** [comment]
I agree that's one of the features I miss most compared to PrusaSlicer.

It's incredibly convenient to have some regularly used models saved directly in the slicer.

**Comment by hliebscher** [comment]
it works .... you have to change the config
![shape](https://user-images.githubusercontent.com/9274162/221266348-d3ea9bf9-85d3-4fc0-9bc1-82919a6a3c0a.jpg)
look at RATRIG file

**Reply by rideorroot** [reply]
What I am talking about is the shape library in Prusa slicer.
You can save stls to a shape library so for example if right clicking > add primitive you can add a benchy or whatever.

**Reply by 11B4P** [reply]
https://help.prusa3d.com/article/shape-gallery_301453

**Reply by 11B4P** [reply]
[cinnamon-2023-02-24T140956-0600.webm](https://user-images.githubusercontent.com/94768479/221281521-102abfa9-01ea-4250-ba7c-d031b2ee5454.webm)

I hope this worked

**Reply by liftbag** [reply]
> What I am talking about is the shape library in Prusa slicer. You can save stls to a shape library so for example if right clicking > add primitive you can add a benchy or whatever.

That's right!


https://user-images.githubusercontent.com/14921654/221432034-7d4f0cfe-f976-4c2a-8006-7414707695c5.mp4
